Serializer/JsonLDNormalizer.php

```diff
@@ -63,7 +64,7 @@ public function supportsNormalization($data, $format = null, array $context = []
 
     public function supportsDenormalization($data, $type, $format = null, array $context = [])
     {
-        return isset($data['@type']) ||
+        return isset($data['@type']) && $type !== stdClass::class ||
```

Comment:
I think this logic needs to be in brackets ensure correct outcome. Do the test not pass without it? If so why is this needed for stdClass but not nullable date

Reply:
Yep the test fails without this line. This logic is necessary to avoid being denormalized via the jsonld serializer. The jsonld denormalize function attempts to find the correct class but will fail to as it no longer exists.

Reply:
But only in this case if the API sends `@type` which it really shouldn't in this case, otherwise the entity should've been documented

Reply:
The Apis will sent a @type attribute for json blobs

Reply:
I would consider that an API bug then. Do we need that for BC?

Reply:
I don't believe we do need that for BC. I have removed the logic from the supports method